.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
See my comment here:

https://github.com/pmacct/pmacct/pull/504#issuecomment-841541647

Please clarify.

Reply:
Hi! Yes, I also try to keep `.dockerignores` as softlinks to `.gitignore` files (as I initially attempted here), unless there are additions/extra lines that justify the copy. And while I was doing the multi-stage changes, I realized there is a line that is very beneficial (which tends to be a quite recurrent scenario) and justifies that copy: 
https://github.com/pmacct/pmacct/pull/504/files#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5R53

Without that, as an example, a change in the last line of your `Dockerfile` invalidates all cached layers for everything after a `COPY . .`, as the Dockerfile itself changes, while docker is smart enough to just invalidate the layers behind the Dockerfile change if the Dockerfile is not in the build context.

Bringing that file change to this PR, as suggested.

Reply:
>Without that, as an example, a change in the last line of your Dockerfile invalidates all cached layers 

I see make sense.

To avoid having to maintain two times the same file (the contents of `.gitignore`), we could create `.dockerignore-base` containing just the extra lines to add on top of `.gitignore`. We would still track `.dockerignore`.

In this way, if a change is done in `.gitignore`, you can easily do:

```
cat .gitignore > .dockerignore
cat .dockerignore-base >> .dockerignore
```

Even the pipeline can have a simple check to make sure `.gitignore` is actually container in the tracked version of `.dockerignore` to avoid getting desynced. Thoughts?

Reply:
Despite I do agree maintaining the duplication is not great (and I tend to avoid things like this as much as possible), I feel the associated overhead is already "assumed" by the community, so this special `.dockerignore` management may be a bit surprising for contributors. But we can implement that check, and maybe consider an alternative: CI checking that the `.dockerignore` contains everything we have in `.gitignore` plus a set of lines from a whitelist (initially just the `Dockerfile` line),  which is the property we are looking for, so we avoid the introduction of that `dockerignore-base` and we reduce the surprise factor.

Reply:
That would be my preference, but let's see what @paololucente thinks. If the decision is to duplicate, then... well, I will learn to live with it :joy: